client/src/api/managment-service.ts

```diff
@@ -182,6 +206,81 @@ class ManagmentServiceApi {
     return user.data;
   }
 
+  public async getReport(id: string): Promise<ReportDetails> {
+    await this.refreshTokenIfExpired();
+    const report = await this.axiosInstance.get(`/api/v1/reports/${id}`);
+    return report.data;
+    // return {
+    //   analyzedApplications: 123,
+    //   analyzedNodes: 20,
+    //   id: 'report-1',
+    //   clusterId: 'cluster-1',
+    //   title: 'Report for this time',
+    //   urgency: 'HIGH',
+    //   totalApplicationEntries: 1000,
+    //   totalNodeEntries: 1000,
+    //   sinceMs: 212414123,
+    //   toMs: 212414123,
+    // };
```

Comment:
I commented out the mock implementation instead of removing it, in case someone doesn't have the managment-service configured properly yet (I intend to remove it once the integration is over)